app/_src/production/secure-deployment/api-server-auth.md

```diff
@@ -301,6 +301,95 @@ kumactl delete global-secret admin-user-token
    {% endtab %}
    {% endtabs %}
 
+### Offline token issuing
+
+In addition to the regular flow of generating signing keys, storing them in secret, and using it to sign tokens on the control plane, Kuma also offers offline signing of tokens.
+In this flow, you can generate a pair of public and private keys and configure the control plane only with public keys for token verification.
+You can generate all the tokens without running the control plane.
+
+The advantages of this mode are:
+* easier, more reproducible deployments of the control plane. More in line with GitOps.
```

Comment:
nitpick: If I remember the docs tutorial correctly entries in lists should look similar so this extra sentence looks a bit off. Maybe we could do:

```suggestion
* easier, more reproducible deployments of the control plane, and more in line with GitOps.
```